pkgs/by-name/hx/hxn/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+  stdenv,
+  darwin,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hxn";
+  version = "1.0.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-zAqYaPtbXqC1YFzCL8EwE1HhuSqVl5lAfnAftwBvnoE=";
+  };
+
+  cargoHash = "sha256-wza/N0+hux5OHnvsLAPwTQLmJqYEr1EjgjpF6k32Yb8=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ];
+
+  meta = with lib; {
+    description = "Blazingly fast tool to grab screenshots of url/webpages from terminal.";
```

`meta.description` should:
 - be factual as well as short and succinct
 - not end with punctuation
For further details please refer to https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes
```suggestion
    description = "Save screenshots of urls and webpages from terminal";
```